### PR TITLE
chore(docs): Added Readme to the project

### PR DESCRIPTION
**Pull request by Blackcipher101**
Fixed #2 
Created Readme Explaining the purpose of the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
...
+ <ul style="list-style-type:disc;">
+  <li>Signup</li>
+  <li>Login</li>
+  <li>Display Products</li>
+   <li>Display Details</li>
+  <li>Add to a cart</li>
+  <li>Calculate total</li>
+  <li>Add inventory</li>
+</ul> 
+<p>This is created Pyhon's Django Framework</p>
```
Comment (yashk2000) [inline README.md:12]: Typo in python.

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
-# Shopping-Cart
+<h1>Shopping cart</h1>
```
Comment (yashk2000) [inline README.md:1]:
```suggestion
<h1>Shopping Cart</h1>
```

File: README.md (hunk 1 above)
Comment (Blackcipher101) [reply README.md:12]: ok, will take care next time ;)

Comment (yashk2000) [reply README.md:12]: You can fix it in this pr itself. No need to wait for a next time :smile:

**Pull request merged**